src/tools/run-make-support/src/rustc.rs

```diff
@@ -85,6 +85,31 @@ impl Rustc {
         self
     }
 
+    /// Specify number of codegen units
+    pub fn codegen_units(&mut self, units: usize) -> &mut Self {
+        self.cmd.arg(format!("-Ccodegen-units={units}"));
+        self
+    }
+
+    /// Specify directory path used for incremental cache
+    pub fn incremental(&mut self, path: impl AsRef<Path>) -> &mut Self {
+        let path = path.as_ref().to_string_lossy();
+        self.cmd.arg(format!("-Cincremental={}", path));
```

Comment:
Could you avoid the lossy conversion and instead construct this argument as an `OsString`?